tests/codebuild/run_test.sh

```diff
@@ -112,3 +112,54 @@ function build_fsx_from_s3()
 
    export FSX_ID=$FSX_ID
 }
+
+function delete_all_tests()
+{
+    # Stop jobs so we can do PrivateLink test.
+    kubectl delete hyperparametertuningjob --all
+    kubectl delete trainingjob --all
+    kubectl delete batchtransformjob --all
+    kubectl delete hostingdeployment --all
+    kubectl delete model --all
+}
+
+# Delete a K8s resource and ensure all AWS resources were cleaned up successfully
+# Parameter:
+#    $1: CRD type
+#    $2: Filename of test
+#    $3: (Optional) Timeout
+# e.g. verify_delete trainingjob xgboost-mnist.yaml ".status.sageMakerTrainingJobName"
+function verify_delete()
+{
```

Comment:
I wanted to ensure that when the training jobs are "running" that we can send them a "stop" signal when the user deletes the k8s resource. If we delete the resource after our tests have completed then all it is doing is deleting it from etcd - it does not make any API calls to SageMaker.

Reply:
I think the confusing thing here is the name of this function. It looks like all other tests we have (`verify_*`) yet performs the role of a helper function. If the name is changed to convey the fact that it is not a test alone and only a helper function, it will become much easier to grasp.

Reply:
maybe `setup_delete_test`?